beam/beam/overrides/stock_entry.py

```diff
@@ -19,6 +19,25 @@ def update_stock_ledger(self):
 		finished_item_row = self.get_finished_item_row()
 		self.get_sle_for_source_warehouse(sl_entries, finished_item_row)
 		self.get_sle_for_target_warehouse(sl_entries, finished_item_row)
+
+		# Add handling_unit to Stock Ledger Entries
```

Comment:
@lauty95 I want to understand why this is required - my thought is that this _should_ be automatically handled by the Inventory Dimension.

Reply:
I read this again, it's my fault for misunderstanding. False alarm. All good.